components/Note/Note.js

```diff
@@ -86,21 +81,13 @@ const Note = (props) => {
     }
   }, [note.reactions.length, auth.pk]);
 
-  useEffect(() => {
-    if (userData) {
-      onUserDataLoad();
-    }
-  }, [userData]);
-
```

Comment:
Was able to get rid of this jankyness. Doesn't seem necessary after moving the profile querying down into the NoteHeader.